packages/analyticsdx-template-lint/src/schemas/layout-schema.json

```diff
@@ -626,6 +627,173 @@
+        "backgroundImage": {
+          "type": "object",
+          "description": "Image to display in the background of this guidance panel. It should be a vertical image and it will be fixed at the bottom of the guidance panel.",
+          "additionalProperties": false,
+          "doNotSuggest": false,
+          "properties": {
+            "name": {
+              "type": "string",
+              "description": "The image name."
+            },
+            "namespace": {
+              "type": ["null", "string"],
+              "description": "The image namespace.",
+              "defaultSnippets": [{ "label": "\"\"", "body": "${0}" }]
+            }
+          },
+          "defaultSnippets": [
+            {
+              "label": "New background image",
+              "body": {
+                "name": "${1:name}"
+              }
+            }
+          ]
+        },
+        "items": {
+          "type": "array",
+          "description": "The set of the items to display in this guidance panel.",
+          "minItems": 0,
+          "items": {
+            "type": "object",
+            "additionalProperties": false,
+            "required": ["type"],
+            "properties": {
+              "type": {
+                "type": "string",
+                "description": "The item type.",
+                "enum": ["Text", "Image", "LinkBox"],
+                "enumDescriptions": ["Displays text.", "Displays an image.", "Displays a link item."]
+              },
+              "text": {
+                "doNotSuggest": true
+              },
+              "variant": {
+                "doNotSuggest": true
+              },
+              "image": {
+                "doNotSuggest": true
+              },
+              "title": {
+                "doNotSuggest": true
+              },
+              "icon": {
+                "doNotSuggest": true
+              },
+              "url": {
+                "doNotSuggest": true
+              }
+            },
+            "anyOf": [
+              {
+                "type": "object",
+                "required": ["type", "text"],
+                "doNotSuggest": false,
+                "additionalProperties": false,
+                "properties": {
+                  "type": { "const": "Text" },
+                  "text": {
+                    "type": "string",
+                    "description": "The text to display in this item.",
+                    "doNotSuggest": false
+                  },
+                  "variant": {
+                    "type": "string",
+                    "description": "The text variant to display.",
+                    "doNotSuggest": false,
+                    "enum": ["SubHeader"],
+                    "enumDescriptions": ["Text is displayed in bolded sub-header format."]
+                  }
+                }
+              },
+              { "$ref": "#/definitions/imageItem" },
+              {
+                "type": "object",
```

Comment:
This is very minor but I think we might as well refactor this under LinkBox object to be under `definitions` and then have a reference to it here (like we are doing w/ `imageItem`)? Also I might be misremembering but I believe I made it so the LinkBox can appear as part of the page's `items` too. Not sure if we allowed or forbade it as part of the GroupBox items though.

Reply:
Ohh I wasn't aware of linkBox is also allowed for panel items. It seems to work for both page's items and groupbox items so I updated the vscode for both to support linkbox :))